packages/node_modules/@ciscospark/widget-recents/README.md

```diff
@@ -104,6 +104,14 @@ To use the space widget within an existing React appliction, a developer can ins
 
   All of the React configurable properties are listed in the ["Configuration"](#Configuration) section.
 
+#### Usage with [Webex SDK](https://developer.webex.com/docs/sdks/browser)
```

Comment:
Is this something you want to add to the TOC?

Reply:
Done